### DISCUSSION TITLE
overlap issue while using position fixed for a card having dropdown menu and datagrid

### DISCUSSION THREAD

**Discussion by tshailesh-vmw** [vmware/clarity#6065] [General] [answered]
Hi Team,

I am facing this issue where dropdown menu comes behind data grid columns. I have a card which have position fixed.
What is the best way to solve this issue. Here is a [stackblitz](
https://stackblitz.com/edit/clarity-v3-light-theme-jleq9z) reproduction of the issue.

Thank you

**Comment by mathisscott** [comment, marked as answer]
Because you are using position fixed on the top element, you will need to go higher in z-index than the datagrid header. I [accomplished that here](https://stackblitz.com/edit/clarity-v3-light-theme-jz3ngy?file=src%2Fapp%2Fapp.component.css) by increasing the z-index of the `.icon` element to `502`.

**Reply by tshailesh-vmw** [reply]
Thank you.
